script/Ciel/Day 10/10_01_CIEL10_4_3.txt

```diff
@@ -387,92 +414,105 @@ Yeah... Setting that aside―――
+ she %{i}is%{/i} an unpredictable walking disaster,
+ so that's honestly not an unreasonable description of her, but...
+ she's not exactly―――"
 }
 [sha:16e6d16ddc85f49d19d84cd946717623a1cda88e]{
 -- Page 271, Offset 27803.
 -- 危うい、なんて事はない筈だ。
-A danger or anything like that, I think.
+She's not exactly %{i}dangerous%{/i}, right?
 }
 [sha:3fc44f272fc3588317a15122569c675c09fc75b6]{
 -- Page 271, Offset 27804.
 -- アイツは意味のない行動はしない。その力はすべて吸血鬼を倒すことに使われていた。
-She won't act senselessly. Her power is entirely dedicated to defeating vampires.
+She doesn't do anything without a good reason to,
+ and the only thing she's done with her powers is defeat vampires.
 }
 [sha:3dbb5612d523dee3fc91c11bf54685dd54b0934a]{
 -- Page 272, Offset 27805.
 -- 「どうでしょうか。今の彼女は自分の感情に従って活動しています。
-"I wouldn't be so sure. As she currently is, she behaves under the influence of her emotions.
+"I wouldn't be so sure.
+ Right now, her actions are completely driven by her emotions.
 }
 [sha:7422c921dd3df7b32a34a85ad32a6da729f45daf]{
 -- Page 272, Offset 27806.
 -- これまでは真祖としての<条約|コード>に従っていたのに、その条約を破りかけている」
-Until now, she's followed a code as a True Ancestor and is now about to break this pact."
+Up to this point she's always followed the True Ancestor's code,
+ but she's suddenly decided to break that pact."
 }
 [sha:9bc18b46851436d2a8c7ac52cb2636f145be491f]{
 -- Page 273, Offset 27807.
 -- 「……アレは無邪気な子供そのものです。
-"She's exactly like a naïve child.
+"She's the very picture of a naïve child.
 }
 [sha:8dfd300e48eaa667adc346bb30b2e94d82d8805c]{
 -- Page 273, Offset 27808.
 -- 恐ろしいのは、その子供が初めて手にしたオモチャが太陽だった……と言えば、現状がどれほど危険なのか理解できると思います」
-What's frightening is the first toy this child has gotten her hands on was the sun... I think that should tell you how dangerous the present situation is."
+Now, imagine how terrifying it would be if the first toy that child got her hands on was the sun...
+ I think that should tell you how dangerous the present situation is."
 }
 [sha:0368b949e00d2051d54fe7f52dc9d1e18505e57c]{
 -- Page 274, Offset 27809.
 -- 先輩は淡々と語ってはいるが、その眼は笑っていなかった。
-Her tone sounds indifferent, but her eyes shows that she's not taking this lightly.
+Her tone is indifferent, but it's clear from the look in her eyes that she's not taking this lightly.
 }
 [sha:e026628638fe4922b609c9fefd0c7e79bff0b299]{
 -- Page 274, Offset 27810.
 -- ……先輩はこう言っている。
-What Senpai is saying―――
+What Senpai's saying is this:
 }
 [sha:5270f09a0734a7b80d628f85b5794d0f92928809]{
 -- Page 274, Offset 27811.
 -- この街の人間にとって吸血鬼はもう脅威ではなくなった。
-Is that vampires represent no threat anymore to the humans of this city.
+To the people of this city, vampires are now the least of their concerns.
 }
 [sha:aff6ddcd004cf904faca833eefa1b00d3293c571]{
 -- Page 274, Offset 27812.
 -- いま祈るべきは、ひとりの子供がその手から太陽を落とさずに去ってくれる幸運だけだと。
-And that one must merely wish to be lucky enough for this one child to leave without dropping the sun.
+Instead, they need to hope against hope that this one child leaves without accidentally dropping the sun.
```

Comment:
This one has a bunch of mistakes. I also feel like 'accidentally' is a bit wrong. 
>Instead, they should pray that this child leaves leaves before it decides to drop the sun in its hands on them.